libnavui-maneuver/src/main/java/com/mapbox/navigation/ui/maneuver/view/MapboxPrimaryManeuver.kt

```diff
@@ -76,18 +77,24 @@ class MapboxPrimaryManeuver : AppCompatTextView {
      * Invoke the method to render primary maneuver instructions
      * @param maneuver PrimaryManeuver
      */
-    fun renderManeuver(maneuver: PrimaryManeuver, roadShields: List<RoadShield>?) {
+    fun renderManeuver(maneuver: PrimaryManeuver, routeShields: Set<RouteShield>?) {
         val exitView = MapboxExitText(context)
-        exitView.setExitStyle(exitBackground, leftDrawable, rightDrawable)
+        // exitView.updateTextAppearance(options.exitOptions.textAppearance)
```

Comment:
Why is this one commented out? Will you add a comment to explain

am noticing the `MapboxSecondaryManeuver` calls it